apps/bare-expo/app.json

```diff
@@ -32,6 +32,9 @@
       "supportsTablet": true,
       "requireFullScreen": false
     },
+    "web": {
+      "bundler": "metro"
```

Comment:
Shouldn't it be the default since we already deprecated webpack?

Reply:
![image](https://github.com/expo/expo/assets/5597580/c68fffd1-7124-4ca7-8056-c4f1e76b016e)
![image](https://github.com/expo/expo/assets/5597580/7d2df721-5b09-460a-b536-7edf190a31dc)

Maybe it should, but it's not – might be something for CLI @byCedric